content/02.intro.md

```diff
@@ -2,11 +2,11 @@
 
 Deep learning (DL), a subfield of machine learning (ML) implementing artificial neural networks with many layers, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
 In many cases, novel biological insights have been revealed through careful evaluation of DL methods ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z].
-However, a lack of relevant up-to-date resources focused on biological applications of DL pose a significant barrier to newcomers wishing to utilize state-of-the-art DL in their research.
+However, a lack of concise recommendations for biological applications of DL hamper newcomers wishing to apply state-of-the-art DL in their research.
 As DL is an active and specialized research area, detailed resources are rapidly rendered obsolete and few resources articulate general DL best practices to the scientific community.
-Most instructional literature focuses on ML broadly, rather than DL specifically, further limiting accessibility and reproducibility.
```

Comment:
Honestly I couldn't really understand why reproducibility was here so I removed it.

Reply:
Yeah, I remember commenting on that sentence elsewhere. I don't think we should say that, and I don't think it's true in this very general way.

Reply:
There's tons of DL instructional literature, just not a lot in bio (that I know of). We should clarify this sentence either way. What about: "Most instructional literature for biologists focuses on ML broadly, rather than DL specifically."

Reply:
Is that true over the past year, or in the general sense of what is in existence? It's also not entirely clear what qualifies as instructional literature. I'm open to having something there, but I'm not sure quite what it should be.

Reply:
What if we drop that sentence (L7) and add the following to L6: "few resources articulate general DL best practices to the scientific community broadly and the biological community specifically"

Reply:
👍  Want me to make that change now?

Reply:
Ahh - looks like you got it! 👍